frontend_tests/node_tests/templates.js

```diff
@@ -1716,3 +1716,40 @@ run_test('recipient_row', () => {
     assert(html.indexOf('<script>alert("Hello")</script>') === -1);
     assert(html.indexOf('&lt;script&gt;alert(&quot;Hello&quot;)&lt;/script&gt;') !== -1);
 });
+
+run_test('invitation_failed_error', () => {
+
+    let data = {
+        error_message: "We weren't able to invite anyone.",
+        error_list: "hamlet@zulip.com: Account has been deactivated.",
+        is_invitee_deactivated: true,
+        check_if_admin: true,
+    };
+
+    let html = render('invitation_failed_error', data);
+    let p = $(html).find("a").first();
+    assert.equal(p.text().trim(), 'translated: organization settings.');
+
+    data = {
+        error_message: "We weren't able to invite anyone.",
+        error_list: "hamlet@zulip.com: Account has been deactivated.",
+        is_invitee_deactivated: true,
+        check_if_admin: false,
+    };
+
+    html = render('invitation_failed_error', data);
+    p = $(html).find("a").first();
+    assert.equal(p.text().trim(), '');
```

Comment:
Check here the content of `<p>` tag is `Organization administrators can reactivate deactivated users.`